webui/react/src/constants/colors.ts

```diff
@@ -21,7 +22,6 @@ export const GLASBEY = [
   [ 197, 89, 14 ],
   [ 67, 135, 23 ],
   [ 0, 211, 145 ],
-  [ 0, 121, 206 ],
```

Comment:
Removing this color because it is fairly close to the newly added color and it's only 15 or so down the list.